src/NeuralNetwork/index.js

```diff
@@ -518,7 +518,7 @@ class DiyNeuralNetwork {
     };
 
     // if debug mode is true, then use tf vis
-    if (this.options.debug === true) {
+    if (this.options.debug) {
```

Comment:
Hmmm. I think any non-empty string would result ina truthy evaluation here, right? I think that if we allow this, we might want to specify either boolean equality or equality with the string 'true', so that doing something like `debug: 'false'` doesn't evaluate to true in this line.

Reply:
Oh, indeed, yeah I hadn't consider that, how about:

```suggestion
    if (this.options.debug === true || this.options.debug === 'true') {
```

Reply:
This looks good to me!